Author: Robert Frohl <rfrohl@suse.com>
Date: Thu Aug 13 10:47:01 2026 +0200

Make building virtualbox-kmp-longterm possible again
under openSUSE with kernel-longterm >= 6.18.44.

References: https://github.com/VirtualBox/virtualbox/issues/812

diff -Nuar a/vboxvideo/vbox_fb.c b/vboxvideo/vbox_fb.c
--- a/vboxvideo/vbox_fb.c	2026-09-15 20:12:51.000000000 +0300
+++ b/vboxvideo/vbox_fb.c	2026-09-15 20:55:29.782971272 +0300
@@ -330,7 +330,7 @@
 		return ret;
 	}
 
-#if RTLNX_VER_MIN(6,19,0) || RTLNX_VER_RANGE(6,12,103, 6,13,0) || RTLNX_RHEL_RANGE(9,9, 9,99)
+#if RTLNX_VER_MIN(6,18,44) || RTLNX_VER_RANGE(6,12,103, 6,13,0) || RTLNX_RHEL_RANGE(9,9, 9,99)
 	info = helper->info;
 #elif RTLNX_VER_MIN(6,2,0) || RTLNX_RHEL_RANGE(8,9, 8,99) || RTLNX_RHEL_RANGE(9,3, 9,99)
 	info = drm_fb_helper_alloc_info(helper);
